json/classic/src/test/java/ch/qos/logback/contrib/json/classic/JsonLayoutTest.java

```diff
@@ -48,11 +49,11 @@ public void addToJsonMap() throws Exception {
         jsonLayout.add("key2", false, "value2", map);
         jsonLayout.add("key3", true, null, map);
 
-        assertTrue(map.size() == 1);
-        assertTrue(map.containsKey("key1"));
+        assertThat(map.size(), is(1));
+        assertThat(map.containsKey("key1"), is(true));
         assertEquals(map.get("key1"), "value1");
-        assertFalse(map.containsKey("key2"));
-        assertFalse(map.containsKey("key3"));
+        assertThat(map.containsKey("key2"), is(false));
+        assertThat(map.containsKey("key3"), is(false));
```

Comment:
This could be improved with:

```java
// import static org.hamcrest.Matchers.hasKey;
// requires dependency on hamcrest-junit
// http://search.maven.org/#artifactdetails%7Corg.hamcrest%7Chamcrest-junit%7C2.0.0.0%7Cjar
assertThat(map, hasKey("key1"));
```

which would output a helpful error like this:

```
java.lang.AssertionError: 
Expected: map containing ["key1x"->ANYTHING]
     got: <{key1=value1}>

Expected :map containing ["key1x"->ANYTHING]
Actual   :<{key1=value1}>
```

Compare that to:

```
java.lang.AssertionError: 
Expected: is <true>
     got: <false>

Expected :is <true>
Actual   :<false>
```